kernel/thread/thread.h

```diff
@@ -324,30 +305,13 @@ void lock_all_threads();
 
 void unlock_all_threads();
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// Thread status
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
 /**
  * Get the status of a thread atomically
```

Comment:
update the docs that this is for easier compat with the C# library, maybe even add a todo to remove this eventually